### ISSUE TITLE
Remove hidden=true from Material Toolbar array component

### ISSUE THREAD

**Issue by danielclariondoor** [eclipsesource/jsonforms#991]
In the Array example, you are showing a delete button, but I can't see how to create the delete button.

**Comment by edgarmueller** [comment]
I'm not exactly sure what you are asking for. Are you looking to hide the delete button (since this is not possible out-of-the-box right now I've created an issue #993)? 
The delete button is created as part of the `TableToolbar` in the `MaterialArrayControlRenderer` if that's what you are looking for (assuming you are using the material-renderers package).

**Comment by danielclariondoor** [comment]
A delete button is not showing up for me at all. 
My schema has, in part, this:
```
 "vehicle": {
            "type": "array",
            "items": {
              "type": "object",
              "properties": {
                "make": {
                  "type": "string",
                  "enum": [
                    {"unknown": "Alpha Romeo"},
                    {"american": "Ford"},
                    {"italian": "Lamborghini"}
                  ]
                },
                "model": {
                  "type": "string",
                  "enum": [
                    "Fill in"
                  ]
                },
                "miles": {
                  "type": "integer"
                }
              }
            }
          }
```
and the result is this:
![screenshot from 2018-06-01 09-55-56](https://user-images.githubusercontent.com/18540311/40853612-9b4e598a-6583-11e8-954c-69dd656d204d.png)

**Comment by edgarmueller** [comment]
This seems weird, there's no toolbar showing up for you, like in the attached screenshot?

![tablepng](https://user-images.githubusercontent.com/711092/40855582-339b584a-65d5-11e8-8db9-2af0366cddc8.png)

**Comment by danielclariondoor** [comment]
No. You can see in my sceenshot.

**Comment by edgarmueller** [comment]
@danielclariondoor I don't see any screenshot. Did you forget to attach it, maybe?

**Comment by danielclariondoor** [comment]
It's a partial screenshot. Right after "and the result is" above. Hard to see until you click on it.
Here's the entire window:
![screenshot from 2018-06-04 09-22-18](https://user-images.githubusercontent.com/18540311/40929180-d774c078-67d8-11e8-84ae-9d18c2c306bd.png)

**Comment by edgarmueller** [comment]
I had a look at this and I think this is likely caused by `<Toolbar hidden={true}>` (which causes `display: none` to be set), from which we don't know how it ended up there, so we'll definitely remove it. The reason that this is working nevertheless is that MUI overrides the the style with `display: flex` and hence it still shows up. Out of curiosity: what MUI version are you using?

**Comment by edgarmueller** [comment]
Fixed via #997
